pkgs/applications/virtualization/kvmtool/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchgit, fetchurl, lib, zlib, libaio, libbfd }:
+
+stdenv.mkDerivation {
+  pname = "kvmtool";
+  version = "2021-12-14";
+
+  src = fetchgit {
+    url = "https://git.kernel.org/pub/scm/linux/kernel/git/will/kvmtool.git";
+    rev = "cdd7d8cc0109bb8e2a0a04c5fe904b5ad4f07a80";
+    sha256 = "0gp0fq130q7m5z1gppan4cd5ip2zxr7hjx4z9aysjy0nj584rihk";
+  };
+
+  # x86: Fixed Unable to execute init process since glibc version 2.33
+  patches = [ (fetchurl {
+    url = "https://github.com/kvmtool/kvmtool/pull/5.patch";
```

Comment:
This url is not immutable, try use links to individual commits such as https://github.com/kvmtool/kvmtool/commit/20c04c3c4936bc168966e01b636e42950fda7628.patch

Reply:
Thank you! Resolved.